src/v2/view-builder/views/SSOExtensionView.js

```diff
@@ -0,0 +1,21 @@
+import { loc } from 'okta';
+import BaseView from '../internals/BaseView';
+import BaseForm from '../internals/BaseForm';
+
+const Body = BaseForm.extend({
+  noButtonBar: true,
+
+  className: 'ion-form sso-extension-challenge',
+
+  title: loc('signin', 'login'),
+
+  initialize () {
+    BaseForm.prototype.initialize.apply(this, arguments);
+    this.add('<div class="spinner"></div>');
```

Comment:
How does this spinner get removed after the API request is done? 

Additionally I think currently the spinner is pretty basic, but we should move this out to some util file and just call a `showSpinner` method from any view that wants to show a spinner. In the future when we have more stuff built into the spinner it might be helpful. We should also have a scss file for the all the styles specific to the spinner. If we cant tackle it now, maybe create a JIRA for it ?

Reply:
It's not just the spinner get removed, once the form submit responds we shows a new view accordingly and the old view is removed. It's just a random example here that the identity is responded thus Identity view is displayed after. JIRA created: https://oktainc.atlassian.net/browse/OKTA-272203.

Reply:
ah, gotcha. Yeah, it might be best to have control over when the spinner is getting removed.